src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FileFields.scala

```diff
@@ -28,6 +49,14 @@ object FileFields {
       arguments = List(ConsignmentIdArg),
       resolve = ctx => ctx.ctx.fileService.getFiles(ctx.arg(ConsignmentIdArg)),
       tags=List(ValidateHasExportAccess)
+    ),
+    Field(
+      "file",
+      FileDetailsType,
+      arguments = List(FileIdArg),
+      resolve = ctx => ctx.ctx.fileService.getFile(ctx.arg(FileIdArg)),
+      //Assumption that only if the user owns the files can they view the file details
+      tags=List(ValidateUserOwnsFiles)
```

Comment:
Will this endpoint be called by the frontend, or is it just a backend task? If it's the latter, could the authorisation be simpler because we just need to check for the file check roles in the token?

Reply:
At the moment the only places where the `getOriginalPath`, which has triggered TDR-874 task is in the:
- tdr-download-files
- tdr-local-aws

So essentially it is just the backend.